Makefile

```diff
@@ -75,10 +94,10 @@ dev-install: _dev-install-ipywidgets
 	npm install
 	npm run bower
 
-dev: KG_IP?=192.168.99.100
+dev: KG_IP?=$$(docker-machine ip $$(docker-machine active))
```

Comment:
The  default becomes dependent on docker machine. But if user sets KG_IP, that code never runs.  (It's late evaled).